docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+
+NOTE: We now have the image, next step is to release the image for which we need to set up the workspace.
+
+== Setting up workspaces to release an FBC application
+
+Two teams work together to release an application:
+
+* *Development team*- The team that develops and supports the application in a pre-production environment.
+* *Managed environment team* - The team that supports the production instance of that application.
+
+To release an application to a managed environment, the development team creates a `ReleasePlan` object in the developer workspace. The `ReleasePlan` object includes a reference to the application that the development team wants to release, along with the namespace and workspace where the application is supposed to release.
+
+=== Creating a ReleasePlan object
+
+.Prerequisites
+
+* You have an existing Development and Managed workspace.
+* You have installed `oc`.
+* You have completed the steps listed in the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/[Getting started in the CLI] page. 
+
+.Procedures
+
+* In the development workspace, create a `release_plan.yaml` object by running: `oc apply -f releaseplan.yaml -n dev`. It adds the required configuration to your workspace that is needed to execute a release pipeline in a managed workspace.
+
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan <1>
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+name: manual-fbc-release <2>
+spec:
+  application: fbc-example <3>
+  displayName: manual-fbc-release 
+  target: managed-release-team-tenant <4>
+----
+<1> The name of the resource that you are creating.
+<2> The name of the release plan.
+<3> The name of the application that you want to deploy to the managed workspace.
+<4> The workspace to which the system deploys the application. This workspace is created by the Managed environment team (for example, your organization’s SRE team)
+
+.Verification
+
+. Browse the application you are creating and select the **Deployements** tab. The **Deployment** tab lists all the `ReleasePlan` objects that you created.
```

Comment:
```suggestion
. Browse to the application you are creating and select the **Deployments** tab. The **Deployment** tab lists all the `ReleasePlan` objects that you created.
```

Reply:
The second sentence here isn't correct. The deployments tab shows how many instances of their app are running on static environments in their namespace. https://issues.redhat.com/browse/RHTAP-164 will add support for viewing and creating ReleasePlans in the UI. The Releases tab lists all attempted releases of their application to the managed workspace, once the releaseplan and corresponding releaseplanadmission are configured and matching.

If they want to validate that a release was successful, they should look at the Releases tab. There's no way to verify in the UI today that the releaseplan was created and correctly points to a releaseplanadmission.